react/components/Table/CheckboxContainer.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import Checkbox from '../Checkbox'
+
+class CheckboxContainer extends PureComponent {
```

Comment:
Use Component instead since the component has a usage like:

```jsx
onClick={() => doSometing(id)}
```
This arrow function will always make the component rerender making the checks by PureComponent more expensive than using Component.

Reply:
Updated.